mir-ci/mir_ci/robot_libraries/WaylandHid.py

```diff
@@ -29,12 +30,15 @@ async def move_pointer_to_absolute(self, x: float, y: float) -> None:
         """Move the virtual pointer to an absolute position within the output."""
         await self.connect()
         self.move_to_absolute(x, y)
+        self.pointer_position = {"x": x, "y": y}
```

Comment:
Let's update in place?

```suggestion
        self.pointer_position |= {"x": x, "y": y}
```

Reply:
TIL